pkg/security/module/module.go

```diff
@@ -359,7 +359,6 @@ func NewModule(cfg *config.Config) (module.Module, error) {
 
 	// custom limiters
 	limits := make(map[rules.RuleID]Limit)
-	limits[sprobe.AbnormalPathRuleID] = Limit{Limit: 0, Burst: 0}
```

Comment:
I would keep this line until we are sure to know 1. the number of event 2. what to do with them ?

Reply:
I think I fixed the bug I was talking about last week. Making a custom deployment in a bit to make sure. If I don't see any abnormal_path, then I think we should remove the rate limiter.